src/dashboard/Dashboard.tsx

```diff
@@ -50,7 +50,7 @@ const EmptyDashboard = () => {
         </Title>
         <EmptyStateBody>{t("introduction")}</EmptyStateBody>
         <Button variant="link" onClick={() => setRealm("master")}>
```

Comment:
```suggestion
        <Button variant="link" onClick={() => setRealm("master")}>
```

Clicking on "realm info" for any of the realms always changes to realm to "master" in the realm dropdown, and takes me to the info page for the master realm.

Reply:
right changed the button to say 'Server info'